org.eclipse.winery.repository.ui/src/app/wineryNamespaceSelector/wineryNamespaceSelector.service.ts

```diff
@@ -18,11 +18,16 @@ export class WineryNamespaceSelectorService {
     constructor(private http: Http) {
     }
 
-    getAllNamespaces(): Observable<NamespaceWithPrefix[]> {
+    getAllNamespaces(all: boolean = false): Observable<NamespaceWithPrefix[]> {
```

Comment:
Please rename the method to `getNamespaces`, otherwise it is a huge WTF why `all` has the Boolean paramter `all´.

![Humorous image of software quality estimation as a count of how many expletives
you shout when reading code](http://www.osnews.com/images/comics/wtfm.jpg)

(Taken from https://github.com/ryanmcdermott/clean-code-javascript)